app/views/repairs/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_for @repair do |f| %>
+    <div>
+   Repair Type: <%= f.text_field :repair_type, class:'form-control', placeholder:'What type of repair will you like done' %>
```

Comment:
To follow best practice, place the plain HTML "Repair Type:" and "Repair Notes:" (below) inside of a form label, similar to [the first example here](https://api.rubyonrails.org/v5.1/classes/ActionView/Helpers/FormHelper.html).